addons/common/XEH_postInit.sqf

```diff
@@ -158,6 +158,9 @@ if (isServer) then {
     ["enableSimulationGlobal", {(_this select 0) enableSimulationGlobal (_this select 1)}] call FUNC(addEventHandler);
 };
 
+["setVectorDirAndUp", {(_this select 0) setVectorDirAndUp (_this select 1)}] call FUNC(addEventHandler);
```

Comment:
Maybe moving this lines upward 6~7 lines would be better, so they are placed along with the rest of similar handlers.